### PR TITLE
Create [2-guides] migrate from Packer to Lazy

### PR DESCRIPTION
**Pull request by al-ce**
Did not embed the video, the thumbnail is an external link to YouTube

### CODE REVIEW

File: contents/2023/Jan/16/2-guides/1-migrate-from-packer-to-lazy.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (phaazon) [inline contents/2023/Jan/16/2-guides/1-migrate-from-packer-to-lazy.md:1]: You can remove the newline here.

**Review by phaazon — CHANGES_REQUESTED** [review]
Thank you! May you squash all your commits, please?

**Comment by elijahmanor** [comment]
Yay, thanks for this! I was dropping by to see if I should contribute this, but see that you all have it covered! @al-ce and @phaazon thank you both for your work!

**Review by phaazon — APPROVED** [review]
Thanks!

**Pull request merged**